cloudferrylib/utils/files.py

```diff
@@ -89,4 +90,19 @@ def _unlink_dir(dirname):
         return "rm -rf {dir}".format(dir=dirname)
     else:
         raise RuntimeError('Wrong dirname %s, stopping' % dirname)
-    return
+
+
+class RemoteTempDir(object):
+    """Creates remote temp dir using `mktemp` and removes it on scope exit"""
+
+    def __init__(self, runner):
+        self.runner = runner
+
+    def __enter__(self):
+        create_temp_dir = 'mktemp -d'
```

Comment:
Ok, what about the format of directory name? After this patch we will have different format. I think it's better to use one format like in GetTempDir. If we don't need prefix - make it the same in all places in the code.